src/settings/project/mod.rs

```diff
@@ -8,50 +8,238 @@ use crate::terminal::emoji;
 use crate::terminal::message;
 
 use std::collections::HashMap;
-use std::convert::TryFrom;
 use std::fs;
 use std::path::{Path, PathBuf};
-use std::str::FromStr;
 
 use log::info;
 
-use config::{Config, Environment, File, Value};
+use config::{Config, File};
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
-pub struct Project {
+pub struct Target {
```

Comment:
What's the difference between Target and Environment?

Reply:
`Environment` is a dummy struct that the toml is parsed into, and `Target` is constructed from merging the base `Manifest` with the parsed `Environment`